docs/tutorials/best-practices/state.md

```diff
@@ -20,7 +20,7 @@ local function Button(
     local isHovering = scope:Value(false)
 
     return scope:New "TextButton" {
-        BackgroundColor3 = scope:Computed(function(use)
+        BackgroundColor3 = scope:Computed(function(use, scope)
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
        BackgroundColor3 = scope:Computed(function(use)
```

Reply:
reverted